bitbots_quintic_walk/include/bitbots_quintic_walk/walk_engine.h

```diff
@@ -28,14 +28,21 @@ namespace bitbots_quintic_walk {
  */
 class WalkEngine : public bitbots_splines::AbstractEngine<WalkRequest, WalkResponse> {
  public:
-  WalkEngine();
+  WalkEngine(const std::string ns);
```

Comment:
```suggestion
  explicit WalkEngine(const std::string ns);
```
If a constructor with a single argument is not marked explicit, it is a converting constructor, i.e. you can write `WalkEngine my_engine = "namespace";` to call the constructor which can be confusing.